Automattic/jetpack
# API: Refactor wpcom/v2 delete transient endpoint logic

#### Changes proposed in this Pull Request:

* Simplifies wpcom/v2 delete transient endpoint logic added in Jetpack 9.7 (https://github.com/Automattic/jetpack/pull/19632).

#### Jetpack product discussion



p3btAN-1pX-p2#comment-13969

#### Does this pull request change what data or activity we track or use?



Nope.

#### Testing instructions:






- Send a DELETE request to a Jetpack site via the console: https://public-api.wordpress.com/wpcom/v2/sites/{site slug or id}/transients/jetpack_connected_user_data_1
- The response should be success: true or success: false
- Send an invalid DELETE request to a Jetpack site via the console, e.g. with a transient name that doesn't exist.
- The response should be: "Sorry, you are not allowed to delete this transient."
